core/src/main/scala/akka/persistence/dynamodb/internal/JournalDao.scala

```diff
@@ -124,8 +128,22 @@ import software.amazon.awssdk.services.dynamodb.model.Update
             .build()
         }.asJava
 
+      val token = {
+        val firstEvent = events.head
+        val uuid = UUID.fromString(firstEvent.writerUuid)
+        val seqNr = firstEvent.seqNr
+        val bb = ByteBuffer.allocate(24)
+        bb.asLongBuffer()
+          .put(uuid.getMostSignificantBits)
+          .put(uuid.getLeastSignificantBits)
+          .put(seqNr)
```

Comment:
This isn't the maximally compact encoding, but 24 bytes will base64 encode into 32 characters.

Reply:
👍🏼 I think this is good. We get everything needed, and always within the character limit.